### ISSUE TITLE
Emoji aren't loading

### ISSUE THREAD

**Issue by iveanissue** [ys-j/YoutubeLiveChatFlusher#2]
This add-on is setting the `src` attribute of emoji in the live chat to an empty string. 

Here is an example:

![image](https://user-images.githubusercontent.com/99297074/153089960-1a7fc99f-c32a-4194-8aba-d9fa8ee99d7d.png)

```html
<span id="message" dir="auto" class="style-scope yt-live-chat-text-message-renderer">
  ASMR
  <img class="emoji yt-formatted-string style-scope yt-live-chat-text-message-renderer" src="" alt="❤" shared-tooltip-text=":red_heart:" id="emoji-9897">
</span>
```

I have confirmed that it is this add-on causing this issue because emoji go back to normal when this add-on is disabled. 

I am using Mozilla Firefox 96.0.3 (64-bit) on Windows 10.

**Comment by ys-j** [comment]
@iveanissue  Thank you for your report.
Could you teach me about add-on configuration ("Emoji Expression" value) and verifiable video's URL with playback time?

**Comment by iveanissue** [comment]
"Emoji Expression" is set to "Show all".

Example: 
https://youtu.be/9FTAlhZD8tM?t=3323

![image](https://user-images.githubusercontent.com/99297074/154815112-52cd57e5-a36e-4179-bbaa-8ec6fda16b27.png)

I see 👏 in the chat instead of [<img src="https://www.youtube.com/s/gaming/emoji/828cb648/emoji_u1f44f.svg" alt="YouTube's clap emoji" height="30">](https://www.youtube.com/s/gaming/emoji/828cb648/emoji_u1f44f.svg).

**Comment by ys-j** [comment]
@iveanissue Thank you for your reply!  I fixed this bug (v0.1.4.2).
Sorry to bother you, but could you confirm if it works well?

**Comment by iveanissue** [comment]
Awesome! The update fixed it. Thank you.
